src/python/visclaw/Iplotclaw.py

```diff
@@ -11,6 +11,7 @@
 """
 
```

Comment:
All these imports, unless absolutely necessary, should be directly imported rather than imported into the global namespace.  In other words `import matplotlib.pyplot as plt` instead of `from matplotlib import pyplot as plt`.  Also, you do not need to check `sys.modules` for `matplotlib`, you can just import it and set the values.  I am not sure if `matplotlib.interactive` needs to be called before `import matplotlib.pyplot` is done though this would not matter in either case.

Reply:
The `sys.modules` check what not of my doing but I agree to the changes.